### ISSUE TITLE
Car Garage Fix

### ISSUE THREAD

**Issue by Heavybob** [ArmaLife/Altis#491]
I would like for an option if a car is blocking a garage shop then have it be auto impounded as a great trolling tactic can be had with parking a car on a popular car buying area.

**Comment by Jawshy** [comment]
This sounds like something that police should be doing. Also, vehicle shops can have multiple spawn points to use so if one spawn is blocked another can be used to spawn the vehicle.

**Comment by Heavybob** [comment]
Its brilliant how people manage to screw up 8 space vehicle spawn car shop.

**Comment by dardothemaster** [comment]
Yeah I agree with Jawshy,this could be an RP-related thing,you could setup 5+ spawnpoints if the players in your server really want to block the spawns... but they'll get tired before finding where's each spawnpoint.

**Comment by danielstuart14** [comment]
we can set something like that: 
 if there isn't any player inside the vehicle, and the vehicle owner isn't near it'll be impounded

**Comment by MrHeavyMetal** [comment]
Why not also run a check to see if there is any cops online and have it work based of that?

**Comment by danielstuart14** [comment]
yep, I'll start it now.

If there's any cop online, it'll send a message to cops instead impounding the vehicle.

**Comment by Heavybob** [comment]
Should make it also impound if no cops are online

**Comment by tkcjesse** [comment]
@Heavybob ... 

Daniel said: If there's any cop online, it'll send a message to cops **instead impounding** the vehicle.

So if there are no cops on it will impound the vehicle....

**Comment by plunkettscott** [comment]
Any idea when this will be released? This is brilliant. I like the idea of it giving the cops something more to do.

**Comment by BoGuu** [comment]
Read this a couple of hours ago, have come up with a solution that incorporates all mentioned, will release it in the morning, time for bed!

**Comment by BoGuu** [comment]
I've decided to optimise what I've done for better performance. Is there any garages/shops in particular that should be monitored? Would somebody be able to supply me a list of their markers?

**Comment by Jawshy** [comment]
> Is there any garages/shops in particular that should be monitored? Would somebody be able to supply me a list of their markers?

@BoGuu, these would likely be the most congested spawns: 

- `carshop1` - Kavala Car Shop NPC
- `car_g_1` - Kavala Garage spawn marker

**Comment by Jawshy** [comment]
@BoGuu, do you have an update on this?

**Comment by BoGuu** [comment]
@Jawshy 

I've ignored it for a little while, I'll find my laptop and finish it off shortly. Thinking about maybe an FSM instead.
